src/PKSim.UI/BootStrapping/ApplicationStartup.cs

```diff
@@ -103,12 +100,8 @@ private static void updateGoDiagramKey()
 
       private SynchronizationContext getCurrentContext()
       {
-         var context = SynchronizationContext.Current;
```

Comment:
@Yuri05 Interesting. With the devepxress update, the context was initialized...they are missing around with global shit. And it would therefore create some weird issues with multi threading.. Fixed